protocol-definitions/custom/Custom.yaml

```diff
@@ -985,6 +989,10 @@ customTypes:
         type: Address
         nullable: false
         since: 2.1
+      - name: autoStepDownWhenLeader
```

Comment:
why duplicated the same info in the CPMember? Can you not get the member info using the uuid and check this field in MemberInfo?